l2-contracts/contracts/L2ContractErrors.sol

```diff
@@ -1,20 +1,35 @@
 // SPDX-License-Identifier: MIT
 pragma solidity 0.8.20;
 
+// 0xcbd9d2e0
```

Comment:
I don't see value in hardcoding it, how to check that is indeed a correct one?

Reply:
hardcoding just allows devs to quickly grep through the code and check when doing local dev